### ISSUE TITLE
New application: mastodon

### ISSUE THREAD

**Issue by panosnethood** [mazi-project/portal#79]
Mastodon, https://joinmastodon.org/, is becoming more and more popular, and its federation model is very well designed.

I think it would be really nice if MAZI Zones could host mastodon instances that could be potentially federated, but also work offline if desired. 

Some complicated syncing combinations also possible :-)

**Comment by mgaved** [comment]
Hi Panos- so this is a 'twitter- equivalent?' 
Added as no.47  in the MAZI project software contenders list.
Is 'complicated syncing combinations' a positive or a negative? :-) I can imagine synching periodically-offline devices can get very complicated but it's also a very valid challenge for us to consider how we address.

**Comment by panosnethood** [comment]
Yes, it is a decentralized self-hosted twitter equivalent. Didn't have time to check how difficult is to self-host but from the (large) number of instances around it seems quite easy. 

As for syncing it could be both positive and negative, and I have different ideas in mind that I could develop in case it is possible to integrate mastodon in the MAZI toolkit.
